packages/core/src/rules/common/spec.ts

```diff
@@ -110,6 +110,15 @@ export const OasSpec: Oas3Rule | Oas2Rule = () => {
             }
           }
         }
+
+        if (propName === 'example' && node.type === 'array') {
+          if (!matchesJsonSchemaType(propValue, node.type, false)) {
+            report({
+              message: `Expected type \`${propName}\` (array) but got \`${propValueType}\`.`,
```

Comment:
This message is invalid, change it to:

```suggestion
              message: `Expected type \`${node.type}\` but got \`${propValueType}\`.`,
```

Reply:
I think this suggestion is good.